analyzer/stun/stun.spicy

```diff
@@ -201,7 +201,7 @@ type MappedAddress = unit(length: uint16) {
 
 # https://tools.ietf.org/html/rfc5389#section-15.6
 type ErrorCode = unit(length: uint16) {
-	: bytes &size=2;
+	: bytes &size=2 &requires=(length >= 4);  # Pre-condition for the whole ErrorCode
```

Comment:
I think this is the only way we currently have to write this. I filed #1051 to improve this.

Reply:
Thanks!